packages/components/progress-stepper/src/ProgressStepper.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { type CommonProps, type MarginProps } from '@contentful/f36-core';
+import { getStyles } from './ProgressStepper.styles';
+
+export interface ProgressStepperProps extends CommonProps, MarginProps {
+  children: React.ReactNode;
+  /**
+   * @default horizontal
+   */
+  orientation?: 'horizontal' | 'vertical';
+  /**
+   * @default number
+   */
+  stepStyle?: 'number' | 'icon';
+  /**
+   * @default 0
+   */
+  activeStep?: number;
+}
+
+function _ProgressStepper(
+  props: ProgressStepperProps,
+  ref: React.Ref<HTMLDivElement>,
+) {
+  const {
+    children,
+    orientation = 'horizontal',
+    stepStyle = 'number',
+    activeStep = 0,
+    testId,
+  } = props;
+  const styles = getStyles();
+  const hydratedTestId =
+    testId || `cf-ui-progress-stepper-${orientation}-${stepStyle}`;
+
+  const stepsToRender = React.Children.toArray(children);
+
+  const renderSteps = () => {
+    const steps = stepsToRender.map((child, index) => {
+      const stepChild = React.cloneElement(child as React.ReactElement, {
+        orientation,
+        stepStyle,
+        activeStep,
+        key: `steps-rendered-${index}`,
+        stepNumber: index,
+      });
+      return stepChild;
+    });
+    return steps;
+  };
+
+  return (
+    <nav
```

Comment:
Thank you for this feedback, we think this is a great idea! Curious if you have an opinion on if the` aria-label `should be static, such as a simple `aria-label='Progress stepper'` or if it would be useful to have it as a dynamic prop to allow the user to dictate the value of the label itself?